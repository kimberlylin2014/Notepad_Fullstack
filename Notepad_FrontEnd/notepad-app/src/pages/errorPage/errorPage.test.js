import  { shallow } from 'enzyme';
import React from 'react';
import ErrorPage from './errorPage.component';

it('expect to render Error Page', ()=> {
    expect(shallow(<ErrorPage />)).toMatchSnapshot()
})


