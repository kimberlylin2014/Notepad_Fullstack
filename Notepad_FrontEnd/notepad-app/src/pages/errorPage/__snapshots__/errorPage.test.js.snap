// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Error Page 1`] = `
<div
  className="ErrorPage"
>
  <h2>
    There was an error on this page.
  </h2>
</div>
`;
